### PR TITLE
Feature/create event page

### PR DESCRIPTION
**Pull request by ovchnkvlad**

### CODE REVIEW

File: server/controllers/eventController.js [hunk 1]
```diff
@@ -80,12 +80,17 @@ exports.createEvent = async (req, res) => {
...
     max_participants,
     min_age,
-    price
+    price,
+    category
   } = req.body;
-  const cover = req.file || null;
+  let cover;
+  if (req.file) {
+    cover = req.file.path;
```
Comment (chooidos) [inline server/controllers/eventController.js]: Look on 154 row, you should do the same or path will be faulty

Comment (ovchnkvlad) [reply server/controllers/eventController.js]: I will fix it, but is it correct when you will write the path into DB with a static URL?

Comment (ovchnkvlad) [reply server/controllers/eventController.js]: Done

File: client/src/events/pages/AddEvent.css [hunk 2]
```diff
@@ -241,3 +241,78 @@ td.rdtYear:hover {
...
+  margin: 0;
+}
+.create__wrapper {
+  background: #fff;
+  padding: 0;
+}
+.map-container {
+  height: 50vh;
+}
+
```
Comment (yushevchuk) [inline client/src/events/pages/AddEvent.css]: Remove space, please.

File: client/src/events/pages/AddEvent.css [hunk 3]
```diff
@@ -241,3 +241,78 @@ td.rdtYear:hover {
...
+  border: none;
+  color: #fff;
+  border-radius: 6px;
+  text-align: center;
+  margin:0 auto;
+}
+#date::-webkit-input-placeholder {
+  color: #fff;
+}
+
```
Comment (yushevchuk) [inline client/src/events/pages/AddEvent.css]: Remove space, please.

File: client/src/events/pages/AddEvent.css [hunk 4]
```diff
@@ -241,3 +241,78 @@ td.rdtYear:hover {
...
+.mt_10 {
+  margin-top: 10px;
+}
+
```
Comment (yushevchuk) [inline client/src/events/pages/AddEvent.css]: Remove space, please.

File: client/src/events/pages/AddEvent.css [hunk 5]
```diff
@@ -241,3 +241,78 @@ td.rdtYear:hover {
...
+.address_label {
+  margin-top: 25px;
+  margin-bottom: 0;
+  text-align: center;
+}
+
```
Comment (yushevchuk) [inline client/src/events/pages/AddEvent.css]: Remove space, please.

**Pull request merged**